miniapps/nurbs/CMakeLists.txt

```diff
@@ -9,33 +9,47 @@
 # terms of the BSD-3 license. We welcome feedback and contributions, see file
 # CONTRIBUTING.md for details.
 
-configure_file(${PROJECT_SOURCE_DIR}/miniapps/nurbs/square-nurbs.mesh
-               ${PROJECT_BINARY_DIR}/miniapps/nurbs/square-nurbs.mesh
-               COPYONLY)
-
-configure_file(${PROJECT_SOURCE_DIR}/miniapps/nurbs/cube-nurbs.mesh
-               ${PROJECT_BINARY_DIR}/miniapps/nurbs/cube-nurbs.mesh
-               COPYONLY)
-
-configure_file(${PROJECT_SOURCE_DIR}/miniapps/nurbs/pipe-nurbs-2d.mesh
-               ${PROJECT_BINARY_DIR}/miniapps/nurbs/pipe-nurbs-2d.mesh
-               COPYONLY)
-
 add_mfem_miniapp(nurbs_ex1
   MAIN nurbs_ex1.cpp
   LIBRARIES mfem)
 
-add_test(NAME nurbs_ex1_ser
-  COMMAND $<TARGET_FILE:nurbs_ex1> -no-vis)
+add_test(NAME nurbs_ex1_r0_o4_ser
+  COMMAND $<TARGET_FILE:nurbs_ex1> -no-vis -r 0 -o 4)
```

Comment:
Can you add all the new tests to the GNU makefile too? I can help with that if you don't feel comfortable with make. 😄

Reply:
I'll have a look and let you know if I am not able to do it.

Reply:
I have added the test, but did not test...

Reply:
Was able to test the serial test and they seemed to work.

Reply:
Thanks! I'll give it a try too.